repo_health/index/static/client/src/components/issue-stats/issue-stats.component.js

```diff
@@ -0,0 +1,45 @@
+/*
+* issue-stats.component.js - (C) Copyright - 2017
+* This software is copyrighted to contributors listed in CONTRIBUTIONS.md.
+*
+* SPDX-License-Identifier: MIT
+*
+* Author(s) of this file:
+* J.Harding
+*
+* Create issue-stats component.
+*/
+
+import template from '../base-component/base.template';
+import BaseStatsComponent from '../base-component/base-stats.component';
+
+const issueStatsComponent = {
+  template,
+  bindings: {issueStatsUrl: '='},
+  controller: class issueStatsComponent extends BaseStatsComponent {
+    stats = null;
+    loadingStats = true;
+    loadingMsg = "Loading issue stats...";
+    numOfStatsSections = 0;
+
+
+    constructor( $http, $filter ) {
+      super($http, $filter);
```

Comment:
The way you are doing it works, if the base has potential to not use the same as the parent. I just like doing ...arguments because it just pipes whatever is in this constructor to the next. But if you take away the base component, this won't matter!